### PR TITLE
feat(VAvatar): integrate roundable mixin

### PR DESCRIPTION
**Pull request by Kawacrepe**
## Description
Integrate roundable mixin inside VAvatar

## Motivation and Context
Asked by john

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [ ] The PR title is no longer than 64 characters.
- [ ] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [ ] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VAvatar/VAvatar.ts [hunk 1]
```diff
@@ -40,6 +43,7 @@ export default mixins(
         minWidth: convertToUnit(this.size),
         width: convertToUnit(this.size),
         ...this.measurableStyles,
+        ...this.roundedClasses,
```
Comment (nekosaur) [inline packages/vuetify/src/components/VAvatar/VAvatar.ts:46]: This should go in `classes` computed, not `styles`.

File: packages/vuetify/src/components/VAvatar/VAvatar.ts [hunk 2]
```diff
@@ -23,7 +27,6 @@ export default mixins(
       type: [Number, String],
       default: 48,
     },
-    tile: Boolean,
   },
 
   computed: {
```
Comment (nekosaur) [inline packages/vuetify/src/components/VAvatar/VAvatar.ts:32]: Remove `v-avatar--tile` line from `classes` computed.

File: packages/vuetify/src/components/VAvatar/VAvatar.ts (hunk 1 above)
Comment (Kawacrepe) [reply packages/vuetify/src/components/VAvatar/VAvatar.ts]: Ty, it wasn't clear for me

File: packages/vuetify/src/components/VAvatar/VAvatar.ts (hunk 2 above)
Comment (Kawacrepe) [reply packages/vuetify/src/components/VAvatar/VAvatar.ts:32]: Done and I also removed the test which focused on v-avatar--tile

**Pull request merged**